Lib/gettext.py

```diff
@@ -159,9 +163,10 @@ def _parse(tokens, priority=-1):
 
     return result, nexttok
 
+
 def _as_int(n):
     try:
-        i = round(n)
+        _ = round(n)
```

Comment:
Actually assignment is not needed here at all.

Leave it as is. I'll remove this function in separate issue.